hadoop-tools/hadoop-aws/pom.xml

```diff
@@ -324,6 +326,19 @@
       </properties>
     </profile>
 
+    <!-- Use the S3 CRT client -->
```

Comment:
this doesn't seem to be working btw, ran `mvn clean verify -Dit.test=ITestAwsSdkWorkarounds -Dtest=none -Pcrt` as I expect ITestAwsSdkWorkarounds to fail when using CRT, but it still passes. Need to look

Reply:
ITestAwsSdkWorkarounds stops looking for the error text with the sdk update patch